### ISSUE TITLE
comb missing from math module

### ISSUE THREAD

**Issue by SaadiSave** [microsoft/pylance-release#293]
## Environment data

-   Language Server version: 2020.8.2
-   OS and version: Ubuntu 20.04 LTS
-   Python version (& distribution if applicable, e.g. Anaconda): 3.8.2

## Expected behaviour

```python
from math import comb
```
Import resolved without errors

## Actual behaviour

![Screenshot from 2020-08-28 09-14-17](https://user-images.githubusercontent.com/53120037/91519360-03292c80-e910-11ea-86f5-95dd4993dc15.png)

But the class works perfectly when used in:
```python
O = inp(9, 0.32, [4, 5])
O.setdist()
print(O.rnddist(4))
```

## Logs

```
None
```

## Code Snippet / Additional information

```python
None
```

**Comment by jakebailey** [comment]
You have a `typeCheckingMode` above "off", so you're seeing these sorts of type errors. The type stubs for `math` do not declare `comb`: https://github.com/python/typeshed/blob/master/stdlib/2and3/math.pyi

I can see that the python docs say it exists in 3.8+: https://docs.python.org/3.8/library/math.html#math.comb

So this is a typeshed bug and would need to be fixed there. The stub contains other things added in 3.8, so it was likely an oversight.

**Comment by erictraut** [comment]
I just submitted a PR to the typeshed repo to add the missing function: https://github.com/python/typeshed/pull/4489

**Comment by SaadiSave** [comment]
Shouldn't pylance automatically index all functions in a package? Why does it require a .pyi file then?

**Comment by SaadiSave** [comment]
Has an update been released to fix the issue?

**Comment by jakebailey** [comment]
No, not yet. We release once a week, typically Wednesday or Thursday.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2020.9.0, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/master/CHANGELOG.md#202090-3-september-2020
